### PR TITLE
add guide for hugo

### PR DESCRIPTION
**Pull request by netbuk**

### CODE REVIEW

**Comment by netbuk** [comment]
Add guide for DokuWiki too

**Comment by SalocinHB** [comment]
In the future, please create one pull request per guide.

File: source/guide_hugo.rst [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+::
+
+ [isabell@stardust ~]$ mkdir -p apps/hugo_render
+ [isabell@stardust ~]$ mkdir hugo_website
+ [isabell@stardust ~]$ 
+ 
+Step 2
+------
+
+To run Hugo from any directory, add the path to Hugo in PATH. To do this, open bashrc with an editor of your choice (e.g. vim, nano) and add the following line at the end:
```
Comment (SalocinHB) [inline source/guide_hugo.rst]: Please use the correct file name and provide a path, e.g. ~/.bashrc

File: source/guide_hugo.rst [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+::
+
+ [isabell@stardust ~]$ nano ~/.bashrc
```
Comment (SalocinHB) [inline source/guide_hugo.rst]: “Editor of your choice” != nano. At least not for everybody.

File: source/guide_hugo.rst [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+ [isabell@stardust ~]$ export PATH=$HOME/apps/hugo_render/hugo:$PATH
+ [isabell@stardust ~]$
+
+Then restart bashrc with source ``~/.bashrc`` to apply the change.
```
Comment (SalocinHB) [inline source/guide_hugo.rst]: Nothing is restarted here

File: source/guide_hugo.rst [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+Step 3
+------
+
+Now change the directory ``~/apps/hugo_render`` and set up Hugo.
+
+::
+
+ [isabell@stardust ~]$ cd apps/hugo_render
+ [isabell@stardust hugo_render]$ wget https://github.com/spf13/hugo/releases/download/hugo_42.23.1_Linux-64bit.tar.gz
```
Comment (SalocinHB) [inline source/guide_hugo.rst]: If you use a dummy version number, mention this and provide a source for the real URL

File: source/guide_hugo.rst [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+ 
+Step 4
+------
+ 
+After Hugo is now installed on the Uberspace, now put on the Hugo side. To do this, switch to the corresponding directory and create the Hugo page there.
+
+::
+ 
+ [isabell@stardust ~]$ cd ~/hugo_website
+ [isabell@stardust hugo_website]$ hugo new site hugo_web
```
Comment (SalocinHB) [inline source/guide_hugo.rst]: this creates output. Please include it.

File: source/guide_dokuwiki.rst [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+Your blog domain needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your `document root`_, then download and configure DokuWiki_.
+
+You will need to enter the following information:
```
Comment (SalocinHB) [inline source/guide_dokuwiki.rst]: Not in this step!

**Comment by SalocinHB** [comment]
The Hugo guide leaves users with a blank page. Please provide a simple how-to for a 'hello world' page.

**Comment by netbuk** [comment]
> The Hugo guide leaves users with a blank page. Please provide a simple how-to for a 'hello world' page.

This problem is explained in step 5. Installing a theme and example files for content.

**Comment by nichtmax** [comment]
Just a quick heads up: Sorry for the long waiting time, we'll get in touch with you soon. We're a small team and sometimes things take longer than we anticipated...

**Comment by luto** [comment]
I merged this manually. Thank you! :tada:

**Pull request closed without merging**